docs/guides/remote-sessions.md

```
@@ -0,0 +1,118 @@
+
+```bash
+aoe remote add linuxbox 'http://linuxbox:8080/?token=abc123'
+```
+
+The name (`linuxbox`) is the label badged onto each row and used as the section header, so keep it short. Letters, digits, `-`, and `_` only.
+
+### 3. Open the TUI
+
+Remote sections appear at the bottom of the sidebar within a few seconds:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)